test/view.cpp

```diff
@@ -28,16 +28,19 @@ TEST_CASE("get_filename_suggestion() normalizes filenames for saving articles",
 	c.set_view(&v);
 
 	// Default case is exclusively ASCII characters. Should never fail.
-	REQUIRE(v.get_filename_suggestion(example_en).compare("Comparing_Visual.txt") ==
+	REQUIRE(v.get_filename_suggestion(
+			example_en).to_locale_string().compare("Comparing_Visual.txt") ==
 		0);
```

Comment:
Instead of converting filepaths to strings and comparing them as strings, it's better to keep them as `Filepath`s and compare with `operator==`:

```c++
REQUIRE(
    v.get_filename_suggestion(example_en)
    ==
    Filepath::from_locale_string("Comparing_Visual.txt"));
```